### PR TITLE
Add report on the PyLadies Snake workshop, 2019-02-02

### PR DESCRIPTION
**Pull request by encukou**
This is taken from an internal event report for the sponsor. Is the blog a good place to post it? :)

### CODE REVIEW

File: content/2019-03-02_pyladies-snake-report.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+On Saturday, February 2, PyLadies Brno organized a “Snake workshop”.
+
+Let me first write a bit about the overall concept.
+For info about this specific event, jump to “The Saturday” below.
+
+
+## PyLadies CZ Beginners' courses
+
+PyLadies CZ is an informal group of people that (among other things) organize
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md]:
```suggestion
[PyLadies CZ](https://pyladies.cz/) is an informal group of people that (among other things) organize
```

File: content/2019-03-02_pyladies-snake-report.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+3-month Python courses for women (mostly beginners).
+These courses have been going on for about 5 years in Brno, and have heavily
+influenced the workshop, so let me write a bit on how they're organized.
+
+The *organizers* are alumni of past courses – the very people that want to
+learn make the courses happen possible.
+They coordinate sign-ups, make sure they have room with Internet, respond
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md]: I'm not sure Internet still spells with capital I in 2019

Comment (honzajavorek) [reply content/2019-03-02_pyladies-snake-report.md]: also line 47 has it with small letter

File: content/2019-03-02_pyladies-snake-report.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+[available]: https://naucse.python.cz/course/pyladies/
+
+![Beginner PyLadies on the workshop, with coaches in the back]({filename}/images/snake2019-20190202_084911.jpg)
+
+
+## The Snake workshop
+
+The full-day event we had on Saturday is a relatively new addition to the
+“curriculum”.
+It's modeled after the very successful Django Girls workshops, where attendees
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md]:
```suggestion
It's modeled after the very successful [Django Girls](https://djangogirls.org/) workshops, where attendees
```

File: content/2019-03-02_pyladies-snake-report.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+The Snake workshop has beginners make a *game* ([Snake]) in just one day.
+
+Similarly to Django Girls workshops, the Snake workshop targets complete
+beginners – prerequisites are on the level of installing Python, unzipping
+an archive or sending an e-mail.
+
+Unlike Django Girls, the Snake workshop tries to make things easier for
+everyone: organizers, teachers, and attendees. Specifically:
+
+* There are less technologies to learn: rather than Python, HTML, CSS and
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md:77]: and HTML templates, which are a mix of everything 🙈

File: content/2019-03-02_pyladies-snake-report.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+  databases, there's Python and a bit of graphics.
+* With a single lecturer rather than small coach-led groups, it's not
+  a problem if a coach doesn't show up.
+  Also, and the coaches can be relative beginners.
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md]: "Also, and" reads strange

File: content/2019-03-02_pyladies-snake-report.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+[Snake]: https://en.wikipedia.org/wiki/Snake_(video_game_genre)
+
+![See the coaches coaching!]({filename}/images/snake2019-20190202_152819.jpg)
+
+
+## The Saturday
+
+The last instance of the Snake workshop happened on Saturday, February 2nd.
+This was the 4th run so far (after the first trial for PyLadies in
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md:106]: "trial for **the** PyLadies" ? but I'm no expert on articles

File: content/2019-03-02_pyladies-snake-report.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+Hradec Králové in March 2018, one for PyLadies Brno in September
+2018, and one open for anyone in October 2018).
+
+For this workshop, Red Hat provided the venue (Plutonium & Neptunium meeting
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md]: link to RH?

File: content/2019-03-02_pyladies-snake-report.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+We spent the morning (well, until 14:00) explaining basic concepts (numbers,
+expressions, strings, functions, conditions, lists, dicts, imports, loops, …)
+and the afternoon with the game.
+Based on the feedback to date, everyone enjoyed the show.
+There were smiles throughout; the coaches were kind and helpful, and I'd say
+the attendees learned as much as they could in a day.
+
+![The importance of good ol' pen & paper]({filename}/images/snake2019-20190202_162002.jpg)
+
+Of course, nothing is perfect – and there's a lot to improve for next time.
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md:130]: **the** next time?

File: content/2019-03-02_pyladies-snake-report.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+Some people were already leaving, so I rushed through the end – which didn't
+help either the attendees or the coaches.
+(For next time, I already have a plan to shuffle the materials around and
+drop 30 minutes of less interesting stuff.)
+
+Also, there was too much food in the end.
+We had a generous sponsorship from Red Hat *and* people ordered their own
+lunches.
+This one is entirely on me as the link between the sponsor and the organizers;
+next time I will be coordinate better.
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md]: "will be coordinate" reads strange

File: content/2019-03-02_pyladies-snake-report.md [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+atmosphere of the workshop.
+
+![Not quite there yet]({filename}/images/snake2019-20190202_173055.jpg)
+
+
+### And as for me...
+
+I think I'm addicted to this workshop.
+
+It ticks the tree factors of intrinsic motivation are autonomy (deciding what
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md]: "It ticks the tree factors of intrinsic motivation are autonomy ..." reads strange

File: content/2019-03-02_pyladies-snake-report.md [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+to teach and what to leave out in a time-limited workshop),
+mastery (working on something that improvement) and purpose (helping
```
Comment (honzajavorek) [inline content/2019-03-02_pyladies-snake-report.md]: "something that improvement" reads strange - perhaps "something that improves"?

**Review by honzajavorek — COMMENTED** [review]
I'd say it's okay to have it here as it is. It is a valuable insight into how PyLadies work, into the workshop, and into the reasons we stopped doing DG workshops as often as we previously did.

**Comment by encukou** [comment]
Thank you for the reviews (even those which I rejected – more eyes are always good!)

The internal report now ended up at [community.redhat.com](https://community.redhat.com/blog/2019/02/building-snakes-to-learn-code/). I added a mention here so things don't look suspicious.

**Comment by encukou** [comment]
@honzajavorek, ping? Could you approve this, or does it need more changes?

**Review by honzajavorek — APPROVED** [review]
Sorry, somehow this slipped from my notifications 😞

**Pull request merged**